### PR TITLE
Add default skin

### PR DESCRIPTION
**Pull request by jwalgran**
These unbranded UI graphics and general about page are a default "skin" for the application. Adding placeholder images allows the application to compile and run without the need to first create these branded assets.

In addition to the assets, the application requires some configuration files to run.

* OpenTreeMap-Info.plist
* Implementation.plist
* OpenTreeMap.entitlements

This pull adds templates for all three of these and documents them in the README.

---

##### Testing

* Clone the repo into a new directory
* Follow the instructions in the README to create the required config files from the templates. (You can copy in an existing, preconfigured `Implementation.plist`)
* Build and run the application

It should compile and run without error.

---

Connects to #294

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,30 +2,139 @@
...
+Required.
+
+The root URL of the OpenTreeMap tiler service connected to the same
+database as the application specified by `apiurl`.
+
+##### urlname
+
+Optional.
+
+This this is set to the `url_name` of an instance, the iOS application
```
Comment (RickMohr) [inline README.md]: "This this" -> "This"

File: README.md [hunk 2]
```diff
@@ -2,30 +2,139 @@
...
-<a href="https://github.com/OpenTreeMap/otm-core">https://github.com/OpenTreeMap/otm-core</a>
+The source code for these services is available is available on Github:
 
-A default skin can be downloaded using the Fabric (http://docs.fabfile.org/) script included with the OpenTreeMap for iOS source.
+- https://github.com/OpenTreeMap/otm-core
+- https://github.com/OpenTreeMap/otm-ecoservice
+- https://github.com/OpenTreeMap/otm-tiler
+
+
+### Configuration files
```
Comment (RickMohr) [inline README.md]: Please consider this update. (note - listing entitlements file first so it doesn't disappear after the long variable list.)

The iOS application requires three configuration files in the `OpenTreeMap` directory:
* `OpenTreeMap.entitlements`
* `OpenTreeMap-Info.plist`
* `Implementation.plist`

The `OpenTreeMap` directory contains templates for all three, and a Fabric command for generating `OpenTreeMap-Info.plist`.

#### OpenTreeMap.entitlements

Comment (RickMohr) [inline README.md]: I don't see a template file for `Implemenation.plist`

**Comment by RickMohr** [comment]
Nice job. Following the doc allows building the app, and it runs.
+1 after doc updates.

File: README.md (hunk 1 above)
Comment (jwalgran) [inline README.md]: Fixed in rebased README b4a7ee2

File: README.md (hunk 2 above)
Comment (jwalgran) [inline README.md]: Fixed in rebased README b4a7ee2

**Pull request merged**